src/com/xilinx/rapidwright/design/ModuleInst.java

```diff
@@ -759,6 +764,80 @@ public void connect(String portName, int busIndex0, ModuleInst other, String oth
         }
     }
 
+    /**
+     * Connects unconnected SitePinInst inputs on the module instance to either GND
+     * or VCC (some reset input use VCC as an input and use the inverter to drive
+     * GND).
+     */
+    public void tieOffUnconnectedInputs(boolean verbose) {
+        for (Port port : getModule().getPorts()) {
+            if (port.isOutPort())
+                continue;
+
+            for (SitePinInst pin : getCorrespondingPins(port)) {
+                if (pin == null) {
+                    System.err.println("ERROR: Problem encountered with finding corresponding SitePinInst on port "
+                            + getName() + "/" + port.getName() + " skipping...");
+                }
+                if (pin != null && (pin.getNet() == null || pin.getNet().getSource() == null)) {
+                    // If a port has a RST inverter, drive it with VCC to get GND
+                    boolean connectToVCC = pin.getName().contains("RST");
+                    if (verbose) {
+                        System.out.println(getName() + "/" + port.getName() + " SitePinInst=[" + pin
+                                + "] has no source, connecting to " + (connectToVCC ? "VCC" : "GND"));
+                    }
+
+                    connectToStaticNet(port.getName(), connectToVCC);
+                }
+                break;
+            }
+        }
+    }
+
+    /**
+     * Connects a port (all of its corresponding SitePinInsts) to either the GND or
+     * VCC net.
+     * 
+     * @param portName     Name of the port to connect to GND or VCC
+     * @param connectToVCC True to connect to VCC, False for GND
+     */
+    public void connectToStaticNet(String portName, boolean connectToVCC) {
+        NetType type = connectToVCC ? NetType.VCC : NetType.GND;
+        Net staticNet = getDesign().getStaticNet(type);
+
+        Port inPort = getPort(portName);
+        if (inPort.isOutPort()) {
+            throw new RuntimeException("ERROR: Attempting to connnect output port " 
+                    + getName() + "/" + portName + " to " + staticNet);
+        }
+            
+        // Connect logically
+        EDIFCell parent = getCellInst().getParentCell();
+        EDIFNet logicalStaticNet = EDIFTools.getStaticNet(type, parent, getDesign().getNetlist());
+        EDIFPortInst portInst = getCellInst().getPortInst(portName);
+        if (portInst == null) {
+            logicalStaticNet.createPortInst(portName, getCellInst());
+        } else {
+            portInst.getNet().removePortInst(portInst);
+            logicalStaticNet.addPortInst(portInst);
+        }
```

Comment:
When `connectToVCC` is true, this would also make the port logically connected to VCC (whereas if you're relying on the inverter, shouldn't it be GND?)